index.js

```diff
@@ -29,7 +29,7 @@ export default function createStore(store = {}, callbacks = {}) {
   }, {});
 
   // We create the main provider by wrapping all the providers
-  storeUtils.Provider = ({ children }) => {
+  storeUtils.Provider = ({ store, callbacks = {}, children }) => {
```

Comment:
You forgot to add a default object to avoid a crash on line 36 if is not defined:

```suggestion
  storeUtils.Provider = ({ store = {}, callbacks = {}, children }) => {
```

And in line 36 I would change this:

```js
const [value, setter] = useState(store[key]);
```

to: 

```js
const [value, setter] = useState(store[key] || defaultStore[key]);
```

And in line 37:

```js
const cb = callbacks[key];
```

to 

```js
const cb = callbacks[key] || defaultCallbacks[key];
```